src/backend/downloadmanager/downloadqueue.ts

```diff
@@ -126,30 +126,46 @@ async function addToQueue(element: DMQueueElement) {
   if (elementIndex >= 0) {
     elements[elementIndex] = element
   } else {
-    const installInfo = await libraryManagerMap[
-      element.params.runner
-    ].getInstallInfo(element.params.appName, element.params.platformToInstall, {
-      branch: element.params.branch,
-      build: element.params.build
-    })
-
-    element.params.size = installInfo?.manifest?.download_size
-      ? getFileSize(installInfo?.manifest?.download_size)
-      : '?? MB'
-
+    const gameInfo = libraryManagerMap[element.params.runner].getGameInfo(
+      element.params.appName
+    )
     if (
-      element.params.runner === 'gog' &&
-      element.params.platformToInstall.toLowerCase() === 'windows' &&
-      installInfo &&
-      'dependencies' in installInfo.manifest
+      !gameInfo?.thirdPartyManagedApp ||
+      !['origin', 'the ea app'].includes(
+        gameInfo.thirdPartyManagedApp.toLowerCase()
+      )
```

Comment:
I wonder if we can set the result of this in the gameInfo object directly since it's calculated like 4 times

something like setting `gameInfo.eaGame = gameInfo.thirdPartyManagedApp && ['origin', 'the ea app'].includes(gameInfo.thirdPartyManagedApp.toLowerCase())`